docs/tools/swift.pod

```diff
@@ -1,30 +1,114 @@
 =pod
 
+=encoding utf8
+
 =head1 NAME
 
-swift - the amazingly new programming language
+swift - General purpose programming language
 
 =head1 SYNOPSIS
 
-B<swift> [-help] [I<input-filename> [I<program-arguments>]]
+To invoke the Swift REPL (Read-Eval-Print-Loop):
+
+=over
+
+B<swift>
+
+=back
+
+To execute a Swift program:
+
+=over
+
+B<swift> program.swift -- <arguments>
+
+=back
+
+To work with the Swift Package Manager:
+
+=over
+
+B<swift> build|package|run|test [options]
+
+=back
+
+To invoke the Swift Compiler:
+
+=over
+
+B<swiftc> [options] <inputs>
+
+=back
+
+The list of supported options is available through the "-help" option:
+
+=over
+
+B<swift> -help
 
-B<swiftc> [B<-emit-object>|B<-emit-assembly>|B<-emit-library>]
-  [-help]
-  B<-o> I<output-file>
-  I<input-filenames>
+B<swift> build -help
 
-The full list of supported options is available via "swift -help" or "swiftc -help".
+B<swiftc> -help
+
+=back
 
 =head1 DESCRIPTION
 
-B<Swift> is a new, high performance systems programming language.  It has a clean
-and modern syntax, and offers seamless access to existing C and Objective-C code
-and frameworks, and is memory safe (by default).
+Swift is a general-purpose programming language built using a modern approach to
+safety, performance, and software design patterns.
+
+The goal of the Swift project is to create the best available language for uses
+ranging from systems programming, to mobile and desktop apps, scaling up to
```

Comment:
Nit here: That comma between "programming" and "to mobile" shouldn't be there. I know you've transposed this text from elsewhere, but nonetheless it's an extraneous comma.